scalableminds/webknossos
# Added more drawings for empty dashboards

PR adds new icons/ drawing and slight redesign for the list views in the dashboard in line with the new branding.

<img width="1610" alt="image" src="https://github.com/scalableminds/webknossos/assets/1105056/4bc40bd1-e935-48f2-820f-080d5a5c957f">

<img width="1623" alt="image" src="https://github.com/scalableminds/webknossos/assets/1105056/ca058254-e65b-46d4-8781-c52c8afe0f78">

<img width="1622" alt="image" src="https://github.com/scalableminds/webknossos/assets/1105056/d5a7fbb2-39da-46c2-aff6-6c210bacf92e">

First time user:
<img width="1606" alt="image" src="https://github.com/scalableminds/webknossos/assets/1105056/aa6665d7-f7d5-4025-a5b4-07b97d78041f">



### URL of deployed dev instance (used for testing):
- https://emptylistdrawings.webknossos.xyz/

### Steps to test:
- Remove all annotations, tasks, and datasets
  - The easiest way is to drop your DB (`yarn refresh-schema`) and rename the orga in your binary data folder
- Check out the empty dashboard
- Upload a dataset -> regular empty list
- Create a new folder -> empty folder list view
- Create an annotation -> regular annotation list
- Create a task -> regular task list

### Issues:
- Contributes to #6739 

------
(Please delete unneeded items, merge only when none are left open)
